torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1933,6 +2037,17 @@ def forward(self, *args: Any, **kwargs: Any) -> Any:
 
             # Start of a forward pass.
             self.training_state = TrainingState_.FORWARD
+            if self._is_root:
```

Comment:
could we add a documentation to let users know this new support?

Reply:
Added